README.md

```diff
@@ -40,15 +42,15 @@ This plugin includes a server plugin, restart JupyterLab if you have followed th
 
 To configure which/how formatters are invoked, see [configuration](https://jupyterlab-code-formatter.readthedocs.io/configuration.html).
 
-----
+---
 
 ## Getting help
 
 I am most responsive on Discord, feel free to ping me in [Python Discord](https://discord.com/invite/python), the [#editors-ide](https://discord.com/channels/267624335836053506/813178633006350366) channel is a suitable place for that.
```

Comment:
Should we drop this? @ryantam626 as you said you will not be able to provide much support for this, it will avoid frustrations for users and mental load for you?

```suggestion
```

Reply:
yeah that would be sensible - i am now a lot less active on discord these days because of my new job.